File: src/main/java/site/gachontable/domain/admin/port/in/AdminRegister.java
package site.gachontable.domain.admin.port.in;

import site.gachontable.presentation.admin.dto.request.AdminRegisterRequest;
import site.gachontable.presentation.shared.dto.response.RegisterResponse;

public interface AdminRegister {

    RegisterResponse execute(AdminRegisterRequest request);
}

File: src/main/kotlin/site/gachontable/domain/admin/port/in/AdminRegister.kt
package site.gachontable.domain.admin.port.`in`

import site.gachontable.presentation.admin.dto.request.AdminRegisterRequest
import site.gachontable.presentation.shared.dto.response.RegisterResponse

interface AdminRegister {
    fun execute(request: AdminRegisterRequest): RegisterResponse
}
